client/app/domain/anycast/domain-dns-anycast.controller.js

```diff
@@ -12,7 +12,8 @@ angular.module('App').controller(
     }
 
     $onInit() {
-      this.domainName = this.$stateParams.productId;
+      this.domainName = this.$scope.ctrlDomain.domain.displayName;
```

Comment:
should it be safer to use `data` from a` setAction` call?

Reply:
Passing the domain as parameter now.